src/Joy4004482.cpp

```diff
@@ -19,18 +19,16 @@
 #include <stdexcept>
 #include "Joy4004482.h"
 //---------------------------------------------------------------------------
-Joy4004482::Joy4004482(IifGPIO *pio, TSettings::SetJoystick* settings)
+Joy4004482::Joy4004482(IifGPIO *pio, TSettings::SetJoystick* settings) :
+	joy{{ nullptr, nullptr, false, 0 }, { nullptr, nullptr, false, 0 }},
```

Comment:
Q: What's difference between `nullptr` and `NULL`?

Reply:
oh, I see, it's new C++11 keyword. But I think that we could put `NULL` macro here a update globals.h to work with nullptr (I'll change it when I come back home).